### PR TITLE
Work branch

### PR DESCRIPTION
**Pull request by srizzuto**
In this branch I'll work throughout the project.

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
### Status: Invalid Code Review Request :x: 

__Hi @srizzuto, I commend you for the great effort put into this project. However, I have to mark the project as invalid due to the following reasons:__

- [ ] Some of your linter checks were not successful: to consider this project ready for review, you need to ensure that all GitHub Action Checks are running successfully. Kindly fix the linter errors.
- [ ] You should ensure that your PR has a descriptive title and summary. 
- [ ] You should also present a descriptive Readme for your project.


Friendly reminder: You should make sure that your repo looks professional before review.
Please take a step back and read a lesson [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/).
Do not forget to apply 3 simple rules from this lesson (it will be easy as you can use a template provided there). This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

**You should only submit for review when you're ready. Do not feel bad: it's okay to mess things up. A great technique that you should employ is to `commit your changes often`. This will keep track of your changes for you and you can always go back to a previous commit if something is messed up in another.**

* * *
**Please, do not open a new Pull Request for subsequent reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

* * * 

**_You can reach out to me on Twitter (@ola_lere), Github(@teekaytech), or slack (@taofeekolalere )_**

Happy Coding :man_technologist:

**Comment by srizzuto** [comment]
All topics have been fixed and presented as required.

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work.  💻 👀 ✨
- This is one of the few projects I've seen completed to this level on the first submission!
- You did an amazing work replicating the original design and placing every element in it's correct position on the page. Well done. Congratulations.
- You even added the sticky bottom footer which is an optional requirement: excellent.

## However, there are some changes required 👀 🧰 🔨 🔧
- Please use a font that corresponds to the original design here:

| Your version | Original page |
| --- | --- |
| image | image |

**Remember**, you only have 3 opportunities to apply changes to this project in order to get it approved. Please apply the changes required in this code review, but also, make sure to go through the original project description to increase your chances of success in the next code review!

### Submit these changes and request a new code review! 😄 👍

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier)🧑‍💻🗨️🧑‍💻
# Happy Coding! 🧠 ⌨️ 🖱️ 🖥️

**Comment by srizzuto** [comment]
The title was changed to a more suitable font!

**Comment by descholar-ceo** [comment]
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

File: README.md [hunk 1]
```diff
@@ -1 +1,64 @@
...
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```
Comment (descholar-ceo) [inline README.md]: [optional] Modify this section to match your project, remove sections that don't apply. I mean you need to explain how we should get this project running on our local environment, and any section that you see is not needed please remove it. But don't let the default text unchanged

File: README.md [hunk 2]
```diff
@@ -1 +1,64 @@
...
+- GitHub: [@githubhandle](https://github.com/srizzuto)
+- Twitter: [@twitterhandle](https://twitter.com/sbstnrzzt)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/srizzuto/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](issues/).
```
Comment (descholar-ceo) [inline README.md]: [optional] Edit this `issues` link, add the working link, this one is broken!

File: README.md [hunk 3]
```diff
@@ -1 +1,64 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](lic.url) licensed.
```
Comment (descholar-ceo) [inline README.md]: [optional] Edit this `license` link, because the current one is broken

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1235 @@
...
+            <div id="icon">
+                <i class="fa fa-search pointer"></i>
+            </div>
+            <div class="spacecosmos">
+                SPACE & COSMOS
+            </div>
+        </div>
+        <div id="middle-nav-container">
+            The New York Times
+        </div>
```
Comment (descholar-ceo) [inline index.html]: [required] This icon should be a link (clickable)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1235 @@
...
+                                </div>
+                            </div>
+                        </div>
+                        <div class="image-title">
+                            NASA's Last Rocket
+                            <div>March 24</div>
+                        </div>
+                    </div>
+                </div>
+            </div>
```
Comment (descholar-ceo) [inline index.html]: [required] Each box (image and text) in this section should be clickable, they should use anchor tag

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @srizzuto  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's because this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
:warning: _I am sorry for that these details were not mentioned during the first review, but they are mandatory to be implemented, work on them carefully and your project will be ready to go_

- [ ] Read my comments on the navbar icon of `The newyork times` and on the images and text in the bottom part of the page that they should be links

# [Optional changes] :yellow_circle: 
_In order to make your repo even more professional, you should implement the changes I suggested you in your readme file, check my comments in your readme file_
___

### :warning: Warning: According to the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561#) you have 1 code review left for this project, make sure that you implement all of the required changes carefully and request another review while you are sure that it's ready. :warning: 

_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Comment by srizzuto** [comment]
Changes made on requirement!

**Comment by descholar-ceo** [comment]
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @srizzuto  , your project is approved :tada: :tada: :tada: 
You have implemented all minimum requirements for this project to mark it as complete, it doesn't need more changes, you can go ahead and merge all of the changes on base branch,

Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Pull request merged**